src/home/WhyBIT.css

```diff
@@ -0,0 +1,64 @@
+.YBITHead{
+    font-family: MetaProRegular;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 52px;    
+    align-items: center;
+    text-align: center;
+    padding-bottom: 3rem;
+    padding-top: 3rem;
+}
+
+.WhyBITflex{
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+    padding:2rem;
+    align-items: center;    
+}
+
+.WhyBITNumber{
+    font-family: Meta Pro;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 64px;
+    line-height: 97.8%;
+    color: #255DAD;
+}
+
+.WhyBITHeading{
+    font-family: Meta Pro;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 42px;
+}
+
+.WhyBITContent{
```

Comment:
@DhaneshShetty , I think you need to add font-weight: 400 here (as per the Figma file) since I don't think we have a default scope for this elsewhere.